src/HermsManagement.py

```diff
@@ -0,0 +1,76 @@
+from discord.ext import commands
+import logging
+import discord
+  
+class HermsManagement(commands.Cog):

```

Comment:
The terminology I've been using is herms=files, hermables=users. So `HermablesManagement` would be a slightly better name